libnavigation-core/api/current.txt

```diff
@@ -329,6 +341,9 @@ package com.mapbox.navigation.core.lifecycle {
 
 package com.mapbox.navigation.core.navigator {
 
+  public final class CacheHandleWrapperKt {
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/issues/6051

Reply:
That's not an issue, it's how Metalava keeps track of Kotlin files - `Kt` as we're including a `typealias` https://github.com/mapbox/mapbox-navigation-android/blob/4238882c4b93545b6d7da34a290a4e2c0d483f99/libnavigation-core/src/main/java/com/mapbox/navigation/core/navigator/CacheHandleWrapper.kt#L6

In any case, as you can see that block is empty, meaning that we're not exposing anything publicly 👍 

Other examples 👀 https://github.com/mapbox/mapbox-navigation-android/blob/827a8774aa367e07961028401064ff8abae29257/libnavigation-core/api/current.txt#L734-L745